### ISSUE TITLE
Title: WCAG 1.3.1,WCAG 4.1.2: Ensures every form element has a label (input[type="checkbox"])

### ISSUE THREAD

**Issue by A11Ybot-OP** [knative/ux#65]
Tags: Accessibility, WCAG 1.3.1, WCAG 4.1.2, label

Issue: Ensures every form element has a label (label - https://accessibilityinsights.io/info-examples/web/label)

Target application: Home - Knative - https://knative.dev/docs/

Element path: .task-list-item:nth-child(1) > input[type="checkbox"]

Snippet: <input type="checkbox" checked="" disabled="">

How to fix: 
Fix any of the following:
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element has no placeholder attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

Environment: Chrome version 97.0.4692.99
